python/kachaka_api/base.py

```diff
@@ -29,17 +29,17 @@ def __init__(self, target: str = "100.94.1.1:26400"):
         self.stub = KachakaApiStub(grpc.insecure_channel(target))
         self.resolver = ShelfLocationResolver()
 
-    def get_robot_serial_number(self) -> str:
+    def get_robot_serial_number(self, *, timeout: float | None = None) -> str:
```

Comment:
In our case of the other source code,  we usually use `Optional[X]` instead of `X | None`. What do you think about it?
```suggestion
    def get_robot_serial_number(self, *, timeout: Optional[float] = None) -> str:
```

Reply:
`| None` is newer syntax introduced by Python 3.10. It's probably common to use `| None` recently, but `Optional` should be used for older environments.

In this case, we can use `| None` since Playground of Kachaka uses Python 3.10.12.

```
$ python3 --version
Python 3.10.12
```

Reply:
I didn't take the repo coding policy into account, but for consistency I'm fine with using `Optional` instead!
I'd like to update the PR along with other issues.